cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -1995,6 +1995,25 @@ func (p *BicepProvider) ensureParameters(
 			}
 		}
 
+		if param.IsAutoGen() {
```

Comment:
Add comment?

// If the parameter is tagged with `{type: "generate"}`, skip prompting. We generate it once, then save to config for next attempts.`.